api/app/src/main/kotlin/packit/security/WebSecurityConfig.kt

```diff
@@ -58,11 +58,32 @@ class WebSecurityConfig(
 
     @Bean
     @Order(2)
-    fun securityFilterChain(
+    fun ottSecurityFilterChain(
         httpSecurity: HttpSecurity,
-        tokenAuthenticationFilter: TokenAuthenticationFilter,
         filterChainExceptionHandler: FilterChainExceptionHandler
+    ): SecurityFilterChain {
```

Comment:
Do we actually need a whole new chain for this? I'd have expected a new entry in the existing one, by doing something like `.requestMatchers("/packets/{id}/files").permitAll()`.

Reply:
Now I'm no longer using this way of configuring the security per route. Instead, our two custom filters, JWTAuthenticationFilter and OTTAuthenticationFilter, check if the requested route is one of those they should apply to (using a shared regex constant).